### DISCUSSION TITLE
wire:ofline does not work when paired with wire:init

### DISCUSSION THREAD

**Discussion by adeiming** [livewire/livewire#3117] [7. Bugs (archived)]
I have 2 livewire components in app.blade, 

1. livewire offline component, 
2. other livewires component, 

if I don't use wire:init on the 2nd component. wire:offline works properly, but otherwise, it doesn't work at all.
please help.

**Comment by austenc** [comment]
Please show us your blade template at least so that we may reproduce this. Without code, we have to make too many assumptions.

Thank you!

**Comment by adeiming** [comment]
sorry my bad, I made a typo

**Comment by joshhanley** [comment]
@adeiming no worries! Closing this.

If you have issues in future, please make sure to provide sample code and what versions of Laravel and Livewire you are using as per the issue template. Thanks!

**Comment by squishythejellyfish** [comment]
👋 Oh Hi! I'm Squishy, the friendly jellyfish that manages Livewire issues.

I see this issue has been closed.

Here in the Livewire repo, we have an "issues can be closed guilt-free and without explanation" policy.

If for ANY reason you think this issue hasn't been resolved, PLEASE feel empowered to re-open it.

Re-opening actually helps us track which issues are a priority.

Reply "REOPEN" to this comment and we'll happily re-open it for you!

(More info on this philosophy here: https://twitter.com/calebporzio/status/1321864801295978497)

**Comment by adeiming** [comment]
REOPEN

**Comment by adeiming** [comment]
this sample is wire:offline not working:

[https://laravelplayground.com/#/snippets/d983ebba-fc8f-49e2-8965-2d03c6b2805b](https://laravelplayground.com/#/snippets/d983ebba-fc8f-49e2-8965-2d03c6b2805b)

**Comment by adeiming** [comment]
@austenc @joshhanley

**Comment by joshhanley** [comment]
@adeiming can you tell me how to use your playground you've setup to demonstrate the issue? I'm not clear on what's happening. Thanks!

**Comment by adeiming** [comment]
if you disconnect the internet, wire: offline should work.

![image](https://user-images.githubusercontent.com/13284072/112920291-d9004d00-9132-11eb-901a-f30e50331e19.png)

but if it's like this, it works

![image](https://user-images.githubusercontent.com/13284072/112920592-7bb8cb80-9133-11eb-8d54-873f6d94e6a0.png)
